tests/remoteInstances/remoteInstancesHelper.js

```diff
@@ -152,6 +157,13 @@ module.exports = {
         password: process.env.AZURE_POSTGRES_PASS,
       },
     },
+    gc: {
+      gc_postgresql: {
+        address: process.env.GCP_SERVER_IP,
+        userName: process.env.GCP_USER,
+        password: process.env.GCP_USER_PASSWORD,
```

Comment:
try using secret here, I have tested it for my changes and it works fine.

Reply:
@puneet0191 should it be general approach for all vars in this helper ? some wrapper ofr class, or whatever works in this framework ?

Reply:
just like this?
`        address: secret(process.env.GCP_SERVER_IP),
        userName: secret(process.env.GCP_USER),
        password: secret(process.env.GCP_USER_PASSWORD),`

Reply:
@denisok yes, will think about a generic approach, not needed for all vars, some variables we use only to check the job which triggered etc